# Para el control del brazo derecho
mmdt_rat20:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  # Position Controllers ---------------------------------------
  joint1_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint1
    pid: {p: 6000.0, i: 0.1, d: 4.0}

  joint2_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint2
    pid: {p: 6000.0, i: 0.1, d: 4.0}

  joint3_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint3
    pid: {p: 6000.0, i: 0.1, d: 4.0}

  joint4_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint4
    pid: {p: 6000.0, i: 0.1, d: 4.0}

  joint5_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint5
    pid: {p: 6000.0, i: 0.1, d: 4.0}

  joint6_manipIzq_position:
    type: effort_controllers/JointPositionController
    joint: manipIzq_joint6
    pid: {p: 6000.0, i: 0.1, d: 4.0}